datanode/gateway/graphql/schema.graphql

```diff
@@ -1784,7 +1794,7 @@ type Market {
     dateRange: DateRange,
     "Pagination information"
     pagination: Pagination
-  ): TradeConnection
+  ): TradeConnection @deprecated(reason: "Use root trades query instead")
```

Comment:
Could you make the reason a bit clearer/easier to understand? Don't think the word "root" is adding a ton.

Reply:
Resolved https://github.com/vegaprotocol/vega/pull/7979/commits/17eaabb6243fb4c169fd2daaf687603388fb7e12